src/main/java/it/pagopa/pn/deliverypush/action/startworkflow/notificationvalidation/AttachmentUtils.java

```diff
@@ -52,8 +55,14 @@ public class AttachmentUtils {
     private final PnSendModeUtils pnSendModeUtils;
     private final AarUtils aarUtils;
     private final NotificationUtils notificationUtils;
+    private final TimelineUtils timelineUtils;
+
+    public enum F24_RESOLUTION_MODE{
```

Comment:
Esternalizziamo questo enum in modo tale da aggiungere anche il javadoc che spieghi il significato di ogni valore. Inoltre rinominiamo in camelCase il nome dell'enum (F24ResolutionMode).